virtool/groups/db.py

```diff
@@ -99,53 +99,50 @@ async def fetch_group_users(db, group_id: str) -> List[UserNested]:
     ]
 
 
-def lookup_minimal_groups(
-    local_field: str = "groups", set_as: str = "groups"
-) -> list[dict]:
+def lookup_group_minimal_by_id(
+    local_field: str = "group", set_as: str = "group"
+) -> List[dict]:
```

Comment:
Is there somewhere where `local_field` and `set_as` arent the default values here?